emcc.py

```diff
@@ -1613,17 +1618,17 @@ def phase_setup(options, state, newargs):
 
   # Wasm SjLj cannot be used with Emscripten EH
   if settings.SUPPORT_LONGJMP == 'wasm':
+    # We error out for DISABLE_EXCEPTION_CATCHING=0, because it is 1 by default
+    # and this can be 0 only if the user specifies so.
+    if not settings.DISABLE_EXCEPTION_CATCHING:
+      exit_with_error('SUPPORT_LONGJMP=wasm cannot be used with DISABLE_EXCEPTION_CATCHING=0')
```

Comment:
This was just moved from below, to make separate testing of the two options.

Reply:
Why not use  `user_settings.get` here too?  and then I think it can also move down to its old location, making the diff smaller.

Reply:
Done.